rust/protocol/src/sealed_sender.rs

```diff
@@ -1060,6 +1422,10 @@ pub fn sealed_sender_multi_recipient_fan_out(data: &[u8]) -> Result<Vec<Vec<u8>>
     Ok(messages)
 }
 
+/// Decrypt the sender's identity from a sealed-sender message in either the v1 or v2 format.
+///
+/// [sealed_sender_decrypt] consumes the output of this method to validate the sender's identity
+/// before decrypting the underlying message.
```

Comment:
"Decrypts the payload of a sealed-sender message in either the v1 or v2 format". We don't really have language around UnidentifiedSenderMessageContents, but it's not just the sender's identity. It has the real message in there too.

Reply:
This makes sense!